auth-api/src/auth_api/models/dataclass.py

```diff
@@ -14,6 +14,7 @@
 """This module holds data classes."""
 
 from typing import List, Optional
+from xmlrpc.client import DateTime
```

Comment:
Not the import you want here.. probably better to use a string format

Reply:
yup changed